### ISSUE TITLE
Some admin interface text not reading out in NVDA

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov#567]
In a few places in the interface, some admin interface text is not reading out in NVDA. I've not been able to replicate this issue in VoiceOver, so I'm not sure if it's something specific to NVDA?

### Creating a subsite page (/node/add/localgov_subsites_page), Banner tab

“No banner added yet”, does not read out on a screen reader.

![image](https://github.com/localgovdrupal/localgov/assets/19519457/059d0a26-4874-4b00-9613-6532f52479b8)

### Uploading a media document (/media/add/document)

The text (shown below) after the field “Add a new file” does not read out to screen reader users.

![image](https://github.com/localgovdrupal/localgov/assets/19519457/c1516338-8a50-498e-ba3e-6779d837fdca)

### Adding a redirect (/admin/config/search/redirect/add)

After Path* the following path is shown, https://preprod.essex-gov.nomensa.xyz/. This does not read out on a screen reader.

![image](https://github.com/localgovdrupal/localgov/assets/19519457/fec1c839-451c-44ed-83e6-ba3b55f13ea3)

## Expected Result

Instructions should read out on a screen reader, all information available to a sighted person should be available to a blind person.
WCAG 2.1 - 3.3.2 Form controls are properly labelled. https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html
